app/views/authenticatedFullUserReactTemplate.scala.html

```diff
@@ -0,0 +1,35 @@
+@import assets.AssetsResolver
+@import com.gu.identity.play.IdUser
+@(title: String, id: String, js: String, user: IdUser)(implicit assets: AssetsResolver)
+
+@scripts = {
+    <script type="text/javascript">
+        window.guardian = window.guardian || {};
+        window.guardian.user = {
+            email: "@user.primaryEmailAddress",
+            @user.publicFields.displayName.map { displayName =>
+            displayName: "@displayName",
+            }
+            @for(fields <- user.privateFields; firstName <- fields.firstName; lastName <- fields.secondName) {
+            firstName: "@firstName",
+            lastName: "@lastName",
```

Comment:
Revisit in the future to set up a standard way of dealing with users inside the redux state.